ProftaakPeriode4/ProftaakPeriode4/Score.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+#include <type_traits>
+
+class Score
+{
+    public:
+        Score();
+        void addScore(int);
```

Comment:
waarom 2 methodes voor het toevoegen / afhalen van de score ? je kan toch ook -10 punten toevoegen of -10 punten afhalen